recipes/mdnsd/all/conandata.yml

```diff
@@ -0,0 +1,12 @@
+sources:
+  "0.8.0":
+    url: "https://github.com/Pro/mdnsd/archive/v0.8.tar.gz"
```

Comment:
This package cannot be named mdnsd since it is a fork from mdnsd

Reply:
Maybe use another version?
e.g. `0.8.0.pro`
This lives outside semantic versioning and is the same as the `xxxx.cci.YYYYMMDD` we're using now.
@uilianries @danimtb @Croydon

Reply:
Maybe use an extra version suffix? e.g. `0.8.0.sol`?
This is the same as the `.cci.YYYYMMDD` suffix we're using for cci packages. It also lies outside of semantic versioning.
wdyt @Croydon @uilianries @danimtb @jgsogo

Reply:
Since it is a fork and the original is still active I don't think that this fork should get the name at all

If for some reasons this fork should be preferred over the upstream, I think we should apply the same rule as when the name would be too generic and use the author as a prefix, so it would be `pro-mdnsd`

But is there really a good reason to prefer this fork over upstream?

Reply:
@pro actually mentioned to eventually merging thinks back upstream some day

https://github.com/conan-io/conan-center-index/pull/1687#discussion_r443524856

Reply:
I checked again the upstream repo, and they still do not support windows builds.

When I added windows support, I had to change quite a lot in the base source, therefore I cannot merge it back.

Maybe you can choose a name like `mdnsd_pro`

Reply:
@Croydon my initial efforts to contribute all the necessary changes back to upstream were not really handled in a good way by the maintainer:

https://github.com/troglobit/mdnsd/pull/1

Windows support was ignired at all.

Therefore I continued in my own fork.

Reply:
I agree with @Croydon , this package should take a different name as the original is still active and they keep merging changes and releasing. Even if it gets a different name like `pro-mdnsd`, it can use the `self.cpp_info.names` to be consumed with the `mdnsd` name.

Reply:
Looking at the FAQ at the wiki and suggestions here, it seems that a prefix is the preferred method to solve this.

Reply:
> 
> 
> Maybe use an extra version suffix? e.g. `0.8.0.sol`?
> This is the same as the `.cci.YYYYMMDD` suffix we're using for cci packages. It also lies outside of semantic versioning.
> wdyt @Croydon @uilianries @danimtb @jgsogo

@madebr what do you mean by saying that it lies outside of semantic versioning? Isn't conan using Major.Minor.Patch as it versioning semantics?